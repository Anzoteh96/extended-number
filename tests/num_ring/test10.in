-345919163301594841046650967275
252004194437153544979827173056
626554264879885871306942979035
