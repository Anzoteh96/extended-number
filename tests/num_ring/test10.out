Enter the number base b:
Enter two numbers n and m:
b is 345919163301594841046650967275
n mod b is 252004194437153544979827173056
m mod b is 280635101578291030260292011760
n + m is 186720132713849734193468217541
n - m is 317288256160457355766186128571
n * m is 123313228009082749330410062735
n ** m is 210124774838845423562490867226
